modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
I have a better idea. We should create the logwriter without proto, to try the store the messages while we don't reach the file, and the logwriter should drive the reopen timer, and should notify log_pipe_notify the affile_dw calling log_pipe_notify if it is elapsed.